src/famfs_lib.c

```diff
@@ -300,6 +354,7 @@ famfs_get_role(const struct famfs_superblock *sb)
 	if (memcmp(&my_uuid, &sb->ts_system_uuid, sizeof(my_uuid)) == 0)
 		return FAMFS_MASTER;
 
+	printf("This node is in client role\n");
```

Comment:
Should drop this printf before we merge it

Reply:
Removed.
I added it so that in certain failure paths, it would give clarity on why an operation failed.